plugins/speculation-rules/helper.php

```diff
@@ -103,7 +103,7 @@ static function ( string $href_exclude_path ) use ( $prefixer ): string {
 					// Also exclude rel=nofollow links, as plugins like WooCommerce use that on their add-to-cart links.
 					array(
 						'not' => array(
-							'selector_matches' => 'a[rel=nofollow]',
+							'selector_matches' => 'a[rel~="nofollow"]',
```

Comment:
I went with `*-` in #1232 - any ideas what the difference is?:

```suggestion
							'selector_matches' => 'a[rel*="nofollow"]',
```

Reply:
Ah found it. `~=` is correct and I'm wrong:

https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

Reply:
Yeah, `*=` would erroneously match `rel="not-nofollow"`, whereas `~=` would only match the entire token.